kernel/include/vx_intrinsics.h

```diff
@@ -221,6 +221,26 @@ inline void vx_fence() {
     __asm__ volatile ("fence iorw, iorw");
 }
 
+//Matrix load
+//Converted instruction type cause destination registers were not getiing blocked otherwise
+inline void mload(unsigned dest, unsigned  addr) 
```

Comment:
How many matrix registers are there?
These instructions look like there are two input registers and one output register (which is possibly one of the input registers).
Is that correct?

Reply:
1 destination, 1 source from ML - source register not relevant. 
1 destination, 1 source from MS - dest register not relevant. 
Immediate value gives the memory address for load and store.